src/mlpack/methods/lars/lars.cpp

```diff
@@ -129,7 +129,7 @@ void LARS::Train(const arma::mat& matX,
   {
     lambdaPath[0] = lambda1;
     Timer::Stop("lars_regression");
-    return;
+    return maxCorr;
```

Comment:
I'm not sure that `maxCorr` is the objective minimized by LARS.  I think that objective is Equation 1.3 from https://projecteuclid.org/download/pdfview_1/euclid.aos/1083178935 .  Maybe I overlooked how `maxCorr` is equivalent to that value?

Reply:
Actually, I was not sure of that. 
I am returning that because in section 2 fig 3(right) of the paper they have plotted absolute current correlations as function of LARS step . In that maximum correlation decreases as we keep training. So I thought it's quite similar to decreasing error hence I thought returning that might be reasonable. Also I went through code and it was training on maxCorr so I thought that returning maxCorr is correct. Also in our code there is no function or something which is calculating that error. 
If there are modifications let me know

Reply:
I think there is a relation between the two---we keep adding variables to the solution until `maxCorr` is sufficently small, so in some sense `maxCorr` captures the amount of `Y` that is not captured by `X*beta`.  There may be some way to convert to something much closer to the mean-squared-error of the regression, but if not I think `maxCorr` can work.  If you'd like to take the time to look into it further, please feel free.